dataCollection/actualData/runArrivals.py

```diff
@@ -1,12 +1,26 @@
 import os
 from dataCollection.actualData.catchFlights import catchArrivals
 from dataCollection.airports.Chicago import Chicago
+from dataCollection.airports.SanDiego import SanDiego
 
-# from dataCollection.airports.SanDiego import SanDiego
 
-outFolder = "11_21_real"
-os.makedirs(os.path.join(os.path.dirname(__file__), outFolder), exist_ok=True)
-os.makedirs(os.path.join(outFolder, "arrivals"), exist_ok=True)
-os.makedirs(os.path.join(outFolder, "debug"), exist_ok=True)
-airport = Chicago()
-catchArrivals(airport, 60 * 60 * 26, outFolder)  # start nov 21 2:30 pm est
+def catchSAN():
+    outFolder = "SAN_11_22"
+    os.makedirs(os.path.join(os.path.dirname(__file__), outFolder), exist_ok=True)
+    os.makedirs(os.path.join(outFolder, "arrivals"), exist_ok=True)
```

Comment:
If you're not running this from the `actualData` directory, these folders will be made in the wrong place. Can you add `os.path.dirname(__file__)` to all of them (or just make it a variable that gets reused)? Same for the lines in `runDepartures.py`.